### DISCUSSION TITLE
Is there a way to install chrome recovery utility when its blocked by admin.

### DISCUSSION THREAD

**Discussion by winghf** [3kh0/ext-remover#339] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I am wondering if there is away to install chrome recovery utility (when its blocked by admin on webstore) btw all my extensions on my chromebook have been removed. Maybe like using terminal is there a way?

**Comment by 3kh0** [comment]
No, just use a home computer, I have not tried this but you might be able to flash it wiht rufus or balenaEtcher, I have not tried it. You could also get someone else to do it, then sell access to the usb stick, ez money making method.

**Reply by winghf** [reply]
i have a usb stick i though we needed chrome recovery utility extension for this to work

**Reply by winghf** [reply]
found out actually i was thing about that but i wont sell the usb stick
ill just sell to downgrade peoples chromebooks ;D

**Reply by 3kh0** [reply]
yeah do that, make sure that they dont tell on you

**Comment by Aka-but-nice** [comment, marked as answer]
#347 or #348
